dbms/src/Debug/astToExecutor.h

```diff
@@ -252,17 +253,20 @@ struct Project : public Executor
 
 struct Join : Executor
 {
-    ASTPtr params;
-    const ASTTableJoin & join_params;
-    Join(size_t & index_, const DAGSchema & output_schema_, ASTPtr params_)
+    tipb::JoinType tp;
+
+    const ASTPtr using_expr_list;
```

Comment:
on_expr_list is more resonable

Reply:
I am afraid it is not. According to https://dev.mysql.com/doc/refman/8.0/en/join.html

```
join_specification: {
    ON search_condition
  | USING (join_column_list)
}
```
What follows `ON` is just a condition expression but what follows USING is a column list.